flake.nix

```diff
@@ -179,10 +179,10 @@
 
         docs = pkgs.stdenv.mkDerivation {
           name = "hls-docs";
-          src = pkgs.lib.sourceFilesBySuffices ./docs [ ".py" ".rst" ".md" ".png" ".gif" ".svg" ];
+          src = pkgs.lib.sourceFilesBySuffices ./. [ ".py" ".rst" ".md" ".png" ".gif" ".svg" ".cabal" ];
```

Comment:
I think it might be safer to use regex so that it does not include markdown files and image files in each plugin/ghcide/...? But I haven't been generated the docs so don't know how long it takes in general.

Reply:
It doesn't matter in practice, this just affects what's hanging around in the nix build, it doesn't get processed. And we're not using this for production usage anyway, it's just for convenience.